app/monitoring/SentryLogging.scala

```diff
@@ -1,31 +1,28 @@
 package monitoring
 
-import ch.qos.logback.classic.{Logger, LoggerContext}
-import com.getsentry.raven.RavenFactory
-import com.getsentry.raven.dsn.Dsn
-import com.getsentry.raven.logback.SentryAppender
-import org.slf4j.Logger.ROOT_LOGGER_NAME
-import org.slf4j.LoggerFactory
-import com.gu.support.config.Stage
-import app.BuildInfo
+import monitoring.SafeLogger._
 
-object SentryLogging {
-  val AllMDCTags = Seq()
-}
+import config.Configuration
+import io.sentry.Sentry
 
-class SentryLogging(dsnConfig: String, stage: Stage) {
-  val dsn = new Dsn(dsnConfig)
-  SafeLogger.info(s"Initialising Sentry logging for ${dsn.getHost}")
-  val tags = Map("stage" -> stage.toString) ++ BuildInfo.toMap
-  val tagsString = tags.map { case (k, v) => s"$k:$v" }.mkString(",")
-  val sentryAppender = new SentryAppender(RavenFactory.ravenInstance(dsn)) {
-    addFilter(SentryFilters.errorLevelFilter)
```

Comment:
I think this means you can delete: https://github.com/guardian/support-frontend/blob/16b54b1f7d02481f387ce09b38c68628cafb800a/app/monitoring/Logging.scala#L53-L61?

Reply:
Ah, yes! Well spotted.